packages/material-ui/src/TextField/TextField.js

```diff
@@ -298,11 +305,11 @@ TextField.propTypes = {
    */
   SelectProps: PropTypes.object,
   /**
-   * Type attribute of the `Input` element. It should be a valid HTML5 input type.
+   * Type attribute of the `input` element. It should be a valid HTML5 input type.
```

Comment:
Should this link to external documentation?

Reply:
Great idea.

Reply:
I'm adding a link for attribute too.